resources/buildstock.rb

```diff
@@ -380,7 +380,7 @@ def evaluate_logic(option_apply_logic, runner, past_results = true)
 
       # Get existing building option name for the same parameter
       if past_results
-        segment_existing_option = values[segment_parameter]
+        segment_existing_option = values[OpenStudio::toUnderscoreCase(segment_parameter)]
```

Comment:
I haven't looked at this in detail, but would it make more sense to apply OpenStudio::toUnderscoreCase to all of the keys inside the `get_values_from_runner_past_results` method instead?

Reply:
All the keys are already underscore cased. I was previously trying to access them with non underscore cased strings.

Reply:
Ah, right. Still waking up this morning apparently.